web_widget_image_url/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2017 - 2018 Modoolar <info@modoolar.com>
+# License LGPLv3.0 or later (https://www.gnu.org/licenses/lgpl-3.0.en.html).
+{
+    "name": "Web Image URL",
+    "summary": "This module provides web widget for displaying image from URL",
+    "category": "Web",
+    "version": "11.0.1.0.0",
+    "license": "LGPL-3",
+    "author": "Odoo Community Association (OCA), Modoolar",
+    "website": "https://www.modoolar.com/",
```

Comment:
should be the URL of the github repo: https://github.com/OCA/web/

Reply:
Done